docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -282,9 +282,22 @@ To add custom .gitignore, add a file with existing [.gitignore rules](https://gi
 
 ### Labels
 
-To add a custom label set, add a file that follows the [label format](https://github.com/go-gitea/gitea/blob/main/options/label/Default) to `$GITEA_CUSTOM/options/label`
+To add a custom label set, add a file that follows the [YAML label format](https://github.com/go-gitea/gitea/blob/main/options/label/Advanced.yaml) to `$GITEA_CUSTOM/options/label`:
+
+```
+labels:
+  - name: "foo/bar"  # name of the label that will appear in the dropdown
+    exclusive: true # whether to use the exclusive namespace for scoped labels. scoped delimiter is /
+    color: aabbcc # hex colour coding
+    description: Some label # long description of label intent
+ ```
+
+The legacy semi-colon delimited file format can be used that follows [label format](https://github.com/go-gitea/gitea/blob/main/options/label/Default) to `$GITEA_CUSTOM/options/label`.
+
 `#hex-color label name ; label description`
 
+Please review [custom labels configuration](https://docs.gitea.io/en-us/labels/)
```

Comment:
```suggestion
For more information on labels, see the [custom labels configuration]({{< relref "doc/usage/labels.en-us.md" >}}).
```

Reply:
Updated my suggestion. (Revised to avoid "labels" twice in the same sentence.)  
Would also be fine with yours as:
```suggestion
For more information, see [how to use labels]({{< relref "doc/usage/labels.en-us.md" >}}).
```